subgraph.yaml

```diff
@@ -125,7 +125,61 @@ dataSources:
 
   # MARKETS
   - kind: ethereum/contract
-    name: Market
+    name: HL Mock DIA 1
+    network: goerli
+    source:
+      address: "0xc8b8c94694cB8f7Aa5A2e7218D841ef492586A03"
+      abi: Market
+    mapping:
+      kind: ethereum/events
+      apiVersion: 0.0.6
+      language: wasm/assemblyscript
+      entities:
+        - Protocol
+        - Bet
+      abis:
+        - name: Market
+          file: ./abis/Market.json
+      eventHandlers:
+        - event: Claimed(indexed address,uint256)
+          handler: handleClaimed
+        - event: MarketOwnershipTransferred(indexed address,indexed address)
+          handler: handleMarketOwnershipTransferred
+        - event: Placed(uint256,bytes32,bytes32,uint256,uint256,indexed address)
+          handler: handlePlaced
+        - event: Settled(uint256,uint256,bool,indexed address)
+          handler: handleSettled
+      file: ./src/market.ts
+
+  - kind: ethereum/contract
+    name: HL Mock DIA 2
```

Comment:
we should only have one mock

Reply:
i just added all the markets that were available so that the subgraph had maximum coverage to index bets placed etc..
![image](https://user-images.githubusercontent.com/79682833/200210894-9475b8e9-a399-424a-9976-ed0d298fc86f.png)

Reply:
they can be removed in favor of one if that's the better approach - which vault would be best in this case?

Reply:
All the current contracts are in the horse.link repo ReadMe 

```Owner: 0x1Ab4C6d9e25Fc65C917aFBEfB4E963C400Fb9814
Horse Link Token 0xd87E8BF1327f10685c9283859A4Eb022ADbe03F9
Mock USDC : 0xaF2929Ed6758B0bD9575e1F287b85953B08E50BC
Mock DIA: 0x70b481B732822Af9beBc895779A6e261DC3D6C8B
LP Token: 0xB678cF41Fec0DF2D4bF69cE0297311B993deE11b
USDC Vault: 0x77049b43746F9DF174829B4AA3931fE0fCF70280
USDC Market: 0xAbA5571aF0cC8Ea36bB4553D6C4B935B0F53E91e
DIA Vault: 0x7e8Aa9bC57CA64Bf3F91fcE3B0A5F740239F8f59
DIA Market: 0xA0f8A6eD9Df461541159Fa5f083082A6f6E0f795
Registry: 0xd90AE997C32EdE8feCe39694460543868Da0d0D1
Market Oracle: 0x592a44ebad029EBFff3Ee4950f1E74538a19a2ea```

Reply:
updating abis and addresses 👍